spark/v3.5/spark/src/test/java/org/apache/iceberg/spark/actions/TestRemoveOrphanFilesAction3.java

```diff
@@ -42,7 +42,7 @@ public void testSparkCatalogTable() throws Exception {
     SparkCatalog cat = (SparkCatalog) spark.sessionState().catalogManager().catalog("mycat");
 
     String[] database = {"default"};
-    Identifier id = Identifier.of(database, "table" + ThreadLocalRandom.current().nextInt(1000));
+    Identifier id = Identifier.of(database, tableName("table"));
```

Comment:
the same thing can happen with `trashFile`. Can you please update those places too?

Reply:
`String trashFile = tableName("/data/trashfile")`. Might be worth updating `tableName(..)` to `randomName(..)` in this case